### PR TITLE
$loggingQueries defaults to true for --pretend flag to output sql queries

### PR DESCRIPTION
**Pull request by knvpk**
As in L4.2 $loggingQueries is defaults to true , so when we run "php artisan migrate --pretend" we have sql quiries logged to console. But in L5.0 $loggingQueries defaults to false, so pretend flag not working correctly. May i know the reason why in L5.0 has $loggingQueries =false; if there is any reason to do that then enable query log $db->enableQuerylog in Illuminate\Database\Connection@pretend to work pretnd flag perfectly. (I also commented the enable query log).

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,4 @@ composer.phar
 composer.lock
 .DS_Store
 Thumbs.db
+.idea
```
Comment (GrahamCampbell) [inline .gitignore]: Remove this please.

File: src/Illuminate/Database/Connection.php [hunk 2]
```diff
@@ -532,6 +532,8 @@ public function transactionLevel()
 	 */
 	public function pretend(Closure $callback)
 	{
+
+//        $this->enableQueryLog();
```
Comment (GrahamCampbell) [inline src/Illuminate/Database/Connection.php]: Fix cs please

**Comment by GrahamCampbell** [comment]
This was surely disabled by default on purpose?

**Comment by pavankumarkatakam** [comment]
Can i keep the $loggingQueries to false.

File: src/Illuminate/Database/Connection.php [hunk 3]
```diff
@@ -532,8 +532,8 @@ public function transactionLevel()
 	 */
 	public function pretend(Closure $callback)
 	{
+        	$this->enableQueryLog();
 		$this->pretending = true;
-
```
Comment (GrahamCampbell) [inline src/Illuminate/Database/Connection.php]: Please this line please.

Comment (GrahamCampbell) [inline src/Illuminate/Database/Connection.php]: Add one after $this->enableQueryLog(); too plz.

**Comment by pavankumarkatakam** [comment]
I think its clear now :-)

File: src/Illuminate/Database/Connection.php [hunk 4]
```diff
@@ -532,8 +532,10 @@ public function transactionLevel()
 	 */
 	public function pretend(Closure $callback)
 	{
+        	$this->enableQueryLog();
+        	
 		$this->pretending = true;
-
+
```
Comment (GrahamCampbell) [inline src/Illuminate/Database/Connection.php]: No trailing whilespace please.

File: src/Illuminate/Database/Connection.php [hunk 5]
```diff
@@ -532,8 +532,10 @@ public function transactionLevel()
 	 */
 	public function pretend(Closure $callback)
 	{
+        	$this->enableQueryLog();
+
```
Comment (GrahamCampbell) [inline src/Illuminate/Database/Connection.php]: No trailing whilespace please.

**Comment by GrahamCampbell** [comment]
Fix the trailing spaces, then squash to one commit please.

**Comment by pavankumarkatakam** [comment]
I dont know whats going on... Is every thing ok now ? @GrahamCampbell

**Comment by GrahamCampbell** [comment]
You seem to have rebased against the master instead of 5.0. This can't be merged as is.

**Comment by pavankumarkatakam** [comment]
What I'm supposed to do now?

**Comment by pavankumarkatakam** [comment]
I have opened with another [pull request] (https://github.com/laravel/framework/pull/7477) with properly commited. please check that  also.

**Pull request closed without merging**